sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1353,6 +1354,8 @@ def deinit(self):
         if self.y_cable_presence[0] is True:
             y_cable_helper.delete_ports_status_for_y_cable()
 
+        del platform_chassis
```

Comment:
To fix LGTM alert, which seems to be incorrect (it thinks this is a local variable, when it is not), suggest trying to replace with:

```python
del globals()['platform_chassis']

```

Reply:
I am hesitant to move to a less readable form of the code due to a bug in the static analyzer. Is there no way to either accept that it is wrong in this case and merge anyway or ask it to ignore this specific case?

Reply:
I'm not a big fan of adding comments for LGTM to ignore lines, but you can look into what they have available and see if you can find something that works, then we can discuss.

Reply:
I understand, I wouldn't want to add a line to the codebase for it either. Can we not accept the warning and merge nonetheless? 

I feel like that is the most elegant solution.

Reply:
While it may be the most elegant solution, it will degrade the LGTM code quality grade, which is currently an A+. I'd rather not introduce false alarms into the code quality score.

While my initial suggestion my not be as readable, I think it is the best compromise. I prefer it to adding comments purely for the LGTM analyzer.

Reply:
Implemented and tested your suggestion @jleveque